tools/roctx.py

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/env python3
+
+import json
+import argparse
+import os
+from sys import argv as sysargs
+import pandas as pd
+from datetime import datetime
+import venv
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Parser for MIGraphX ROCTX Markers")
+    parser.add_argument('--json-path',
+                        type=str,
+                        metavar='json_path',
+                        help='path to json file')
+    parser.add_argument('--out',
+                        type=str,
+                        metavar='out',
+                        help='output directory for run')
+    parser.add_argument(
+        '--study-name',
+        type=str,
+        metavar='study-name',
+        help='study name is used for naming the output CSV file.')
+    parser.add_argument('--repeat',
+                        type=int,
+                        metavar='repeat',
+                        help='defines number of runs',
+                        default=2)
+    parser.add_argument('--parse', default=False, action='store_true')
+    parser.add_argument('--clean',
+                        default=False,
+                        action='store_true',
+                        help='Removes temporary paths')
+    parser.add_argument('--run',
+                        type=str,
+                        metavar='run',
+                        help='enables run and fetches run configs.')
+    parser.add_argument('--debug', default=False, action='store_true')
+
+    args = parser.parse_args()
+    return args
+
+
+def parse(file):
+    args = parse_args()
```

Comment:
I think we should avoid repeatedly calling `parse_args()` where it's not necessary. In this function, we could just add an extra parameter `debug=False`, and then pass in `args.debug` when calling `parse()` in main.

Reply:
Made args global, let me know what you think.